all_words_gen/src/main.rs

```diff
@@ -99,7 +99,7 @@ fn main() -> Result<(), String> {
 }
 
 fn create_all_words_table(igen: &PlsInflectionGenerator) -> Result<(), String> {
-    let query = r#"DROP TABLE IF EXISTS _all_words; CREATE TABLE _all_words (inflection TEXT NOT NULL, stem_id INTEGER NOT NULL);"#;
+    let query = r#"DROP TABLE IF EXISTS _all_words; CREATE TABLE _all_words (inflectionIndex INTEGER PRIMARY KEY, inflection TEXT NOT NULL, stem_id INTEGER NOT NULL);"#;
```

Comment:
where are we inserting the data?

i am assuming the original mismatches are solved.

Reply:
No explicit insertion for index ('integer primary key' does it).
The others are inserted [here](https://github.com/digitalpalitools/pali-language-services/blob/6809c4dae1b2dc6dd36306ef547185629acfa4a6/all_words_gen/src/inflection_sql_queries.rs#L18).